test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -15,10 +15,10 @@
 import torch
 import torch.nn.functional as F
 from torch import Tensor
-from torch.onnx import symbolic_helper, utils
+from torch.onnx import OperatorExportTypes, symbolic_helper, utils
```

Comment:
```suggestion
import numpy as np
from torch.onnx import OperatorExportTypes, symbolic_helper, utils
```

Add the numpy import in the right position (Github editor does not let me suggest changes to lines that were not updated by the PR)